Source/player.cpp

```diff
@@ -600,7 +600,7 @@ bool PlrHitMonst(Player &player, Monster &monster, bool adjacentDamage = false)
 	}
 
 	if (gbIsHellfire && HasAllOf(player._pIFlags, ItemSpecialEffect::FireDamage | ItemSpecialEffect::LightningDamage)) {
-		int midam = player._pIFMinDam + GenerateRnd(player._pIFMaxDam - player._pIFMinDam);
+		int midam = player._pIFMinDam + GenerateRnd(player._pIFMaxDam - player._pIFMinDam + 1);

```

Comment:
`int midam = player._pIFMinDam + GenerateRnd((player._pIFMaxDam - player._pIFMinDam) + 1);`

While the order of operations here will result in the same result regardless if you add these parentheses or not, it's best practice to make the order of operations explicit, for maintainability and readability, since max - (min + 1) would produce a different result